### PR TITLE
Docker log collection

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
New section that explains how to collect logs from docker container

### Motivation
Agent 6 can now collect docker logs so we need to have this feature documented

### CODE REVIEW

File: content/logs/index.md [hunk 1]
```diff
@@ -107,6 +107,32 @@ logs:
...
+logs:    
+   - type: docker
+     image: httpd    #or label: mylabel:mylabelvalue
+     service: nginx
+     source: nginx
+     sourcecategory: http_web_access
+
+{{< /highlight >}}
+
+If the agent is containerized, please see [here](https://github.com/DataDog/docker-dd-agent#configuration-files) how to mount the YAML configuration files to the agent container.
```
Comment (irabinovitch) [inline content/logs/index.md]: mesos & k8s instructions?

Comment (NBParis) [reply content/logs/index.md]: @irabinovitch  If you can give me a link on installation instruction for agent 6 on mesos&K8s i'll be happy to add them here.

**Comment by NBParis** [comment]
Pending docker log collection release to open the pull request.

**Comment by NBParis** [comment]
@l0k0ms I let you review this

File: content/logs/index.md [hunk 2]
```diff
@@ -107,6 +107,32 @@ logs:
 
 {{< /highlight >}}
 
+### Docker log collection
+
+Agent 6 is able to collect logs from containers. It can be installed [on the host](https://github.com/DataDog/datadog-agent/blob/master/docs/beta/upgrade.md) or [in a container](https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent).
+
+To start collecting logs for a given container filtered by image or label, you need to update the log section in an integration or custom yaml file.
```
Comment (l0k0ms) [inline content/logs/index.md]: - you need to update the log section in an integration or custom yaml file.

+ Update the integration log section in its yaml file, or create a [custom yaml file](/logs/#custom-log-collection)

File: content/logs/index.md (hunk 1 above)
Comment (l0k0ms) [inline content/logs/index.md]: No please.

See [here](https://github.com/DataDog/docker-dd-agent#configuration-files) how to mount the YAML configuration files to the agent container if your agent is containerized.

Comment (NBParis) [reply content/logs/index.md]: 👍

File: content/logs/index.md (hunk 2 above)
Comment (NBParis) [reply content/logs/index.md]: 👍

**Pull request merged**